packages/package.njk

```diff
@@ -74,3 +74,45 @@ eleventyComputed:
   </li>
   {% endif %}
 </ul>
+
+<script src="https://cdn.jsdelivr.net/npm/marked/marked.min.js"></script>
+<script src="https://cdn.jsdelivr.net/npm/dompurify/dist/purify.min.js"></script>
+
+<div id="md"></div>
+
+<script>
+const cacheKey = `md:{{ pkg.readme }}`;
+const ttl = 60 * 60; // 1 hour in seconds
+
+const target = document.getElementById('md');
+const cached = JSON.parse(sessionStorage.getItem(cacheKey) || "null");
+const now = Math.floor(Date.now() / 1000);
+if (cached && (now - cached.time) < ttl) {
+  target.innerHTML = cached.html;
+} else {
+  fetch('{{ pkg.readme }}')
+    .then(res => {
+      if (!res.ok) throw new Error(`HTTP error ${res.status}`);
+      return res.text();
+    })
+    .then(md => {
+      if (DOMPurify.isSupported) {
+          const html = marked.parse(md);
+          const safe = DOMPurify.sanitize(html);
+          sessionStorage.setItem(cacheKey, JSON.stringify({ html: safe, time: now }));
+          target.innerHTML = safe;
+        } else {
+          const escaped = md
+            .replace(/&/g, '&amp;')
+            .replace(/</g, '&lt;')
+            .replace(/>/g, '&gt;');
+          target.innerHTML = escaped;
+          target.className = 'raw';        }
```

Comment:
```suggestion
        const html = marked.parse(md);
        const safe = DOMPurify.sanitize(html);
        sessionStorage.setItem(cacheKey, JSON.stringify({ html: safe, time: now }));
        target.innerHTML = safe;
      } else {
        const escaped = md
          .replace(/&/g, '&amp;')
          .replace(/</g, '&lt;')
          .replace(/>/g, '&gt;');
        target.innerHTML = escaped;
        target.className = 'raw';
      }
```

Just some whitespace. Nice though 👍🏻

Reply:
Damn, I f** up the white-space.  2 spaces aren't enough for me.